benchopt/utils/dependencies_mixin.py

```diff
@@ -94,6 +94,16 @@ def install(cls, env_name=None, force=False):
         """
         is_installed = cls.is_installed(env_name=env_name)
 
+        # Making conda the default command
+        if not hasattr(cls, "install_cmd"):
+            cls.install_cmd = DependenciesMixin.install_cmd
+        else:
+            if cls.install_cmd not in ["conda", "shell"]:
+                raise ValueError(
+                    f"{cls.install_cmd} is not a valid install command. "
+                    "Please use 'conda' or 'shell' as install command."
+                )
```

Comment:
Any class that derive from `DependenciesMixin will have `install_cmd = "conda"`, except if it is overridden, so they should all have a `install_cmd` attribute`.
```suggestion
        if cls.install_cmd not in ["conda", "shell"]:
            raise ValueError(
                f"{cls.install_cmd} is not a valid install command. "
                "Please use 'conda' or 'shell' as install command."
            )
```